onnxruntime/core/mlas/lib/convsym.cpp

```diff
@@ -500,8 +508,7 @@ MlasConvSym(
     }
 
     MLAS_CONV_SYM_POST_PROCESS_PARAMS PostProcessParams = {};
-
-    MlasConvSymSetOutputZeroPoint(PostProcessParams, Params.OutputZeroPoint, Params.InputIsSigned);
+    MlasConvSymSetOutputZeroPoint(PostProcessParams, OutputZeroPoint, Params.InputIsSigned);
```

Comment:
is this done repeatedly on the same set of parameters? should we consider moving this out in the future? Need to change MLAS interface to do that. Since MLAS is not public yet maybe ok?